modules-local/aerodyn/src/AeroDyn_Driver_Subs.f90

```diff
@@ -209,10 +210,10 @@ subroutine Set_AD_Inputs(iCase,nt,DvrData,AD,errStat,errMsg)
 
    real(ReKi)                                  :: z             ! height (m)
    !real(ReKi)                                  :: angle
-   real(ReKi)                                  :: theta(3)
-   real(ReKi)                                  :: position(3)
-   real(ReKi)                                  :: orientation(3,3)
-   real(ReKi)                                  :: rotateMat(3,3)
+   real(R8Ki)                                  :: theta(3)
```

Comment:
What is the difference between `R8Ki` and `DbKi`?

Reply:
`DbKi` is determined by the `DOUBLE_PRECISION` compiler flag in SingPrec.f90:
```
#ifndef DOUBLE_PRECISION
INTEGER, PARAMETER              :: ReKi           = SiKi                        !< Default kind for floating-point numbers
INTEGER, PARAMETER              :: DbKi           = R8Ki                        !< Default kind for double floating-point numbers
                                                  
INTEGER, PARAMETER              :: BYTES_IN_REAL  = BYTES_IN_SiKi               !< Number of bytes per ReKi number     - use SIZEOF()
INTEGER, PARAMETER              :: BYTES_IN_DBL   = BYTES_IN_R8Ki               !< Number of bytes per DbKi number     - use SIZEOF()
#else
INTEGER, PARAMETER              :: ReKi           = R8Ki                        !< Default kind for floating-point numbers
INTEGER, PARAMETER              :: DbKi           = QuKi                        !< Default kind for double floating-point numbers

INTEGER, PARAMETER              :: BYTES_IN_REAL  = BYTES_IN_R8Ki               !< Number of bytes per ReKi number     - use SIZEOF()
INTEGER, PARAMETER              :: BYTES_IN_DBL   = BYTES_IN_QuKi               !< Number of bytes per DbKi number     - use SIZEOF()
#endif
```